### ISSUE TITLE
add proper links and video for aaddons features

### ISSUE THREAD

**Issue by selul** [Codeinwp/woocommerce-product-addon#20]
Right now they are canceled in inc/arrays.php. It can be viewed here /wp-admin/admin.php?page=ppom&view=addons

**Comment by mskapusuz** [comment]
@selul 

currently the url can be accessible by clicking **All Addons**

I think you already bring back the addons with this commit: https://github.com/Codeinwp/woocommerce-product-addon/commit/9f9bbffc61073a80254706abb360bc6fc8e52cd6

But I couldn't understand the expected case of this issue. Could you give more details?

> Right now they are canceled in inc/arrays.php.

I can't understand what you mean here. You meant before your commit?

> add proper links and video for aaddons features

You mean a new landing page for addons? or prepare a landing page/video for each of them? currently all of them are redirected to https://themeisle.com/plugins/ppom-pro/upgrade/?utm_source=wpadmin&utm_medium=alladdonspage&utm_campaign=woocommercepackageprice&utm_content=ppom-pro

-----

**current:**
<img width="1728" alt="Screen Shot 2022-11-14 at 23 31 59" src="https://user-images.githubusercontent.com/25361626/201760013-55033ee6-206c-4b77-afc1-d0745216c60a.png">
<img width="1413" alt="Screen Shot 2022-11-14 at 23 31 33" src="https://user-images.githubusercontent.com/25361626/201760027-8d15cdcf-10ff-4134-95c7-dce8f4de20c8.png">

**Comment by selul** [comment]
@mskapusuz yes, but some of the cards had some video links, or demos which are missing, right now all of them are linking to upgrade page. 

Is not bad as it is, but if we have already content covered in docs or youtube videos,  we can add a another link there.

**Comment by mskapusuz** [comment]
Notes:

* I didn't do anything Google Map Picker in the free, pro repos. Therefore I updated its card and removed texts Google Map Picker related ones. before: https://vertis.d.pr/i/qZpDhb after:https://vertis.d.pr/i/Q0UqZI
